modules/acm/default.auto.tfvars

```diff
@@ -0,0 +1,5 @@
+# This file is included by default in terraform plans
+
+enabled = false
+
+name = "acm"
```

Comment:
please move `name` to YAML config (e.g. `catalog/acm.yaml` vars).
we had issues in Spacelift when we define vars in many diff places

Reply:
@Benbentwo  you removed the `name` from the file, but did you add it to YAML config for `acm`?